manim/cli/project/templates/Graph.py

```diff
@@ -0,0 +1,16 @@
+class Graph(GraphScene):
```

Comment:
We are in the process of moving away from `GraphScene`. It would be great if this template could also use a construction with `Axes` instead.

Reply:
I'm sorry, do you mean ThreeDAxes or setup_axes?

Reply:
I am sorry i did not know that GraphScene was being deprecated. I removed GraphTemplate and added AxesTemplate

Reply:
There was no way for you to know, don't worry about it. :-)